python/lsst/ts/standardscripts/auxtel/latiss.py

```diff
@@ -0,0 +1,244 @@
+            The exposure time for the image, in seconds.
+        shutter : `bool`
+            Should activate the shutter? (False for bias and dark)
+        image_seq_name : `str`
+            A string to identify the image.
+        latiss_filter : `None` or `int` or `str`
+            Filter id or name. If None, ignore setting filter.
+        latiss_grating : `None` or `int` or `str`
+            Grating id or name.  If None, ignore setting grating.
+        latiss_linear_stage : `None` or `float`
+            Linear stage position.  If None, ignore setting linear stage.
+        science : `bool`
+            Mark image as science (default=True)?
+        guide : `bool`
+            Mark image as guide (default=False)?
+        wfs : `bool`
+            Mark image as wfs (default=False)?
+
+        Returns
+        -------
+        endReadout : `SALPY_ATCamera.ATCamera_logevent_endReadoutC`
+
+        """
+
+        await self.setup_atspec(latiss_filter=latiss_filter,
+                                latiss_grating=latiss_grating,
+                                latiss_linear_stage=latiss_linear_stage)
+
+        return await self.expose(exp_time=exptime, shutter=shutter, image_seq_name=image_seq_name,
+                                 science=science, guide=guide, wfs=wfs)
+
+    async def expose(self, exp_time, shutter, image_seq_name,
+                     science=True, guide=False, wfs=False):
+        """Encapsulates the take image command.
+
+        This basically consists of configuring and sending a takeImages
+        command to the camera and waiting for an endReadout event.
+
+        Parameters
+        ----------
+        exp_time : `float`
+            The exposure time for the image, in seconds.
+        shutter : `bool`
+            Should activate the shutter? (False for bias and dark)
+        image_seq_name : `str`
+            A string to identify the image.
+        science : `bool`
+            Mark image as science (default=True)?
+        guide : `bool`
+            Mark image as guide (default=False)?
+        wfs : `bool`
+            Mark image as wfs (default=False)?
+
+        Returns
+        -------
+        endReadout : `SALPY_ATCamera.ATCamera_logevent_endReadoutC`
+
+        """
+        async with self.cmd_lock:
+            # FIXME: Current version of ATCamera software is not set up to take
+            # images with numImages > 1, so this is fixed at 1 for now and we
+            # loop through any set of images we want to take. (2019/03/11)
+            self.atcam.cmd_takeImages.set(numImages=1,
+                                          expTime=float(exp_time),
+                                          shutter=bool(shutter),
+                                          imageSequenceName=str(image_seq_name),
+                                          science=bool(science),
+                                          guide=bool(guide),
+                                          wfs=bool(wfs)
```

Comment:
cmd_takeImages could accept the enum mentioned above, but assuming you'd rather use the XML you have now this could read as:

    science=image_type == ImageType.SCIENCE,
    ....

Reply:
That is a very good idea. I'll see what we can come up with the camera folks.

Reply:
I was talking to the camera folks about this. Actually those flags are to switch on/of the readout of some of the camera CCDs. If wfs=False it will switch off readout of the WFS detectors. Same for guide and science. So, in fact, you can actually switch off the science detector and read the wfs and/or guiding alone...

Reply:
Interesting. So a simple enum isn't a good match. A Flag enum would be fine (since values can be or-d together) or just use the booleans you have, as you prefer.